actions/zendesk/actions.py

```diff
@@ -75,7 +75,7 @@ def get_ticket_comments(
 
     Args:
         zendesk_credentials: Zendesk OAuth2 credentials
-        ticket_id: The ticket ID to pull comments for
+        ticket_id (str): The ticket ID to pull comments for
```

Comment:
The argument list should not state types as they (and return types) are already declared in the method signature. This same comment applies to all similar changes in this pull request.